admin/resources/js/components/PoolDashboard.tsx

```diff
@@ -54,7 +56,28 @@ const routes: Routes<RouterResult> = [
   {
     path: "/classifications",
     action: () => ({
-      component: <ClassificationTableApi />,
+      component: (
+        <div>
+          <Link href="/classifications/create" title="">
+            Create Classification
```

Comment:
I know this work was started before the [i18n PR](https://github.com/GCTC-NTGC/gc-digital-talent/pull/254) was merged into the `main` branch, but you might want to merge main into this branch and add anything new that needs to be translated so that there aren't missing strings down the road.